vuepress/docs/next/docs/compose/app-builder.md

```diff
@@ -0,0 +1,37 @@
+---
+sidebarDepth: 2
+---
+
+# Entando App Builder
+
+The Entando App Builder plays a key role in the composition of Entando Applications. It is a feature-rich, low-code user interface to customize and manage components. It also offers a Dashboard and a modular CMS to streamline the design and build process. 
+
+<img src="./img/app-builder.png" width="800" height="393.76">
+
+The App Builder allows you to create pages and content, configure widgets and plugins, and interact with the [Entando Component Repository](ecr-overview.md) (ECR). It also interfaces with the [Entando Hub](https://hub.entando.com), where you can share components and collaborate.
+
+In the default deployment, the App Builder is a React JS application served by Node. In a quickstart environment, the App Builder is deployed as a container. It is the frontend of the core application and uses REST APIs to communicate with the core instance and Entando Component Manager (ECM). The App Builder can query the ECM to fetch information on Entando Bundles available to the ECR.
+
+### Key Features:
+
+
+* Install component bundles from the Entando Component Repository
+
+* Append or update applications
+
+* Use micro frontends to build pages from modular, editable content
+
+* Deploy standalone packaged business capabilities
+
+* Deliver standardized UX design with page and content templates 
+
+
```

Comment:
how about a picture of the page editing section, where you have the list of components to add on the side. I always liked that capability